main.py

```diff
@@ -21,3 +21,7 @@ async def down(self):
     async def get_tailscale_state(self):
         result = not subprocess.call(["tailscale", "status"], timeout=10)
         return result
+
+    async def get_tailscale_devices(self):
+        result = subprocess.getoutput("tailscale status | awk -v name=2 -v status=5 -v status2=6 -v status3=7 '{print $name,$status,$status2,$status3}'")
```

Comment:
Couple ideas to make this more structured, and so easier to interpret by the UI:

1. Use `tailscale status --json`, parse it on Python: This would allow us to use an intermediary data structure to store devices data
2. Use https://github.com/pulumi/pulumi-tailscale: You get much richer interaction with Tailscale which opens the door for many more features, but adds a new dependency (and the issues that come along: updating, testing, etc) and complicates setup (we'll need to come up with a way for users to provide their Tailscale API key)